Cocktails9/Cocktails9/View/Authentication/RegisterView.swift

```diff
@@ -51,17 +53,23 @@ struct RegisterView: View {
             return
         }
         
-        let newUser = User(email: email, username: username, password: password)
+        // Create new user with empty favorite cocktails list
+        let newUser = User(email: email, username: username, password: password, favoriteCocktails: [])
         
         UserManager.saveUser(newUser)
         
+        LoginView.currentEmail = newUser.email
```

Comment:
It's not a good practice to call properties from other views. I think `AppDataManager` could also handle this.
```suggestion
        AppDataManager.shared().user = newUser.email
```